Purchases/Purchasing/Purchases.swift

```diff
@@ -463,114 +462,127 @@ extension Purchases {
     }
 
     /**
-     * Subscriber attribute associated with the email address for the user
+     * Subscriber attribute associated with the email address for the user.
      *
-     * - Parameter email: Empty String or nil will delete the subscriber attribute.
+     * - SeeAlso: [Subscriber attributes](https://docs.revenuecat.com/docs/subscriber-attributes)
+     * - Parameter email: Empty String or `nil` will delete the subscriber attribute..
      */
     @objc public func setEmail(_ email: String?) {
         subscriberAttributesManager.setEmail(email, appUserID: appUserID)
     }
 
     /**
-     * Subscriber attribute associated with the phone number for the user
+     * Subscriber attribute associated with the phone number for the user.
      *
-     * - Parameter phoneNumber: Empty String or nil will delete the subscriber attribute.
+     * - SeeAlso: [Subscriber attributes](https://docs.revenuecat.com/docs/subscriber-attributes)
+     * - Parameter phoneNumber: Empty String or `nil` will delete the subscriber attribute..
      */
     @objc public func setPhoneNumber(_ phoneNumber: String?) {
         subscriberAttributesManager.setPhoneNumber(phoneNumber, appUserID: appUserID)
     }
 
     /**
-     * Subscriber attribute associated with the display name for the user
+     * Subscriber attribute associated with the display name for the user.
      *
-     * - Parameter displayName: Empty String or nil will delete the subscriber attribute.
+     * - SeeAlso: [Subscriber attributes](https://docs.revenuecat.com/docs/subscriber-attributes)
+     * - Parameter displayName: Empty String or `nil` will delete the subscriber attribute..
      */
     @objc public func setDisplayName(_ displayName: String?) {
         subscriberAttributesManager.setDisplayName(displayName, appUserID: appUserID)
     }
 
     /**
-     * Subscriber attribute associated with the push token for the user
+     * Subscriber attribute associated with the push token for the user.
      *
-     * - Parameter pushToken: nil will delete the subscriber attribute.
+     * - SeeAlso: [Subscriber attributes](https://docs.revenuecat.com/docs/subscriber-attributes)
```

Comment:
These `SeeAlso` don't seem to be working correctly for now. It looks like maybe Xcode doesn't support them anymore? 
I'm going to take a stab at fixing these in a separate PR, but for now, we're just following the standard notation